File: azure-website/GtfsService/Entities/Raw/GtfsStop.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Serialization;

namespace GtfsBackend.Entities.Raw
{
    public class GtfsStop : BaseGtfsEntity<int>
    {
        [XmlAttribute]
        public int Id;
        [XmlAttribute]
        public int Code;
        [XmlAttribute]
        public double Latitude;
        [XmlAttribute]
        public double Longitude;

        [XmlAttribute]
        public string Name;
        [XmlAttribute]
        public string Address;
        [XmlAttribute]
        public string StreetAddress;
        [XmlAttribute]
        public string Town;

        public override int GetId()
        {
            return Id;
        }
    }
}

File: android-app/app/src/main/java/com/tahanot/entities/GtfsStop.java
package com.tahanot.entities;

import org.json.JSONException;
import org.json.JSONObject;
import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Root;

@Root
public class GtfsStop {
	@Attribute
	public int Id;
	@Attribute
	public int Code;
	@Attribute
	public double Latitude;
	@Attribute
	public double Longitude;

	@Attribute
	public String Name;
	@Attribute
	public String Address;
	@Attribute
	public String StreetAddress;
	@Attribute
	public String Town;
	
	public GtfsStop(JSONObject jsonObject) throws JSONException {
		Code = jsonObject.getInt("Code");
		Latitude = jsonObject.getDouble("Latitude");
		Longitude = jsonObject.getDouble("Longitude");
		Address = jsonObject.getString("Address");
		Name = jsonObject.getString("Name");
	}
	
	public GtfsStop() {	
	}
}
